controllers/common/create_tree.go

```diff
@@ -54,9 +54,11 @@ func CreateTrillianTree(ctx context.Context, displayName string, trillianURL str
 	adminClient := trillian.NewTrillianAdminClient(conn)
 	logClient := trillian.NewTrillianLogClient(conn)
 
-	tree, err := client.CreateAndInitTree(ctx, req, adminClient, logClient)
+	deadline := time.Now().Add(time.Duration(120000) * time.Millisecond)
```

Comment:
It can be simplified to
```suggestion
	deadline := time.Now().Add(2 * time.Minute)
```